Creating draft 2024 call for editors (#34)
openjournals/blog.joss.theoj.org: _posts/2024-10-09-call-for-editors.md

---
layout: post
date: 2024-10-09 00:00
title:  "Call for editors"
category:
- announcements
---

Once again, we're looking to grow our editorial team at JOSS!

> Since our launch in [May 2016](https://www.arfon.org/announcing-the-journal-of-open-source-software), our [existing editorial team](http://joss.theoj.org/about#editorial_board) has handled nearly 4500 submissions (~2700 published at the time of writing, 340 under review) and the demand from the community continues to be strong. JOSS now consistently publishes a little over one paper per day, and we see no sign of this demand dropping.

Reviewer: So we're rejected 1/3 of our submissions? This seems high, but maybe so...

> Since our launch in [May 2016](https://www.arfon.org/announcing-the-journal-of-open-source-software), our [existing editorial team](http://joss.theoj.org/about#editorial_board) has handled nearly 4500 submissions (~2700 published at the time of writing, 340 under review) and the demand from the community continues to be strong. JOSS now consistently publishes a little over one paper per day, and we see no sign of this demand dropping.

Reviewer: I don't think the "a little over" is right. We're at about 1.3 - 1.5 papers per day, so I might say "40-45 papers per month, and this is gradually increasing"

New editors at JOSS are asked to make a minimum 1-year commitment, with additional years possible by mutual consent. As some of our existing editorial team are reaching the end of their term with JOSS, the time is right to bring on another cohort of editors.

### Background on JOSS

If you think you might be interested, take a look at our [editorial guide](https://joss.readthedocs.io/en/latest/editing.html), which describes the editorial workflow at JOSS, and also some of the reviews for [recently accepted papers](https://github.com/openjournals/joss-reviews/issues?utf8=%E2%9C%93&q=is%3Aclosed+label%3Aaccepted+). Between these two, you should be able to get a good overview of what editing for JOSS looks like.

> Further background about JOSS can be found in our [PeerJ CS paper](https://peerj.com/articles/cs-147/), which summarizes our first year, and our Editor-in-Chief's [original blog post](https://www.arfon.org/announcing-the-journal-of-open-source-software), which announced the journal and describes some of our core motivations for starting the journal.

Reviewer: ```suggestion
Further background about JOSS can be found in our [PeerJ CS paper](https://doi.org/10.7717/peerj-cs.147), which summarizes our first year, and our Editor-in-Chief's [original blog post](https://www.arfon.org/announcing-the-journal-of-open-source-software), which announced the journal and describes some of our core motivations for starting the journal.
```

More recently we've also written in detail about [the costs related with running JOSS](https://blog.joss.theoj.org/2019/06/cost-models-for-running-an-online-open-journal), [scaling our editorial processes](https://blog.joss.theoj.org/2019/07/scaling), and [talked about the collaborative peer review that JOSS promotes](https://www.youtube.com/watch?v=niRiyaErqwQ).

## How to apply

Firstly, we especially welcome applications from prospective editors who will contribute to the diversity (e.g., ethnic, gender, disciplinary, and geographical) of our board. 

✨✨✨ If you're interested in applying please fill in this [short form](https://forms.gle/SwNqHd3KYi1rwPMEA) by 9th November 2024. ✨✨✨

## Who can apply

Applicants should have significant experience in open source software and software engineering, as well as expertise in at least one subject/disciplinary area. Prior experience with peer-review and open science practices are also beneficial.  

We are especially interested in recruiting editors in the following disciplines, although submissions from all disciplines are welcome:

> - Applied Mathematics

Reviewer: ```suggestion
- Both Pure and Applied Mathematics
```

- Bioinformatics
- Computational Chemistry
- Computer Science
- Ecology
- Economics
- Geology
- Image Processing
- Language Processing

> - Physics and Engineering

Reviewer 2: I think we need to break this down a little or add some subfields - together these are extremely broad. At minimum, lets specify energy systems and fluid dynamics

Author: OK, I updated them.

- Remote Sensing

The JOSS editorial team has a diverse background and there is no requirement for JOSS editors to be working in academia.

### Selection process

The [JOSS editorial team](http://joss.theoj.org/about#editorial_board) will review your applications and make their recommendations. Highly-ranked candidates will then have a short (~30 minute) phone call/video conference interview with the editor(s)-in-chief. Successful candidates will then join the JOSS editorial team for a probational period of 3 months before becoming a full member of the editorial team. You will get an onboarding “buddy” from the experienced editors to help you out during that time.

> _Dan Foreman-Mackey, Olivia Guest, Daniel S. Katz, Kevin M. Moerman, Kyle E. Niemeyer, Arfon M. Smith, Kristen Thyng_

Reviewer: Add Sam?

More recently we've also written in detail about [the costs related with running JOSS](https://blog.joss.theoj.org/2019/06/cost-models-for-running-an-online-open-journal), [scaling our editorial processes](https://blog.joss.theoj.org/2019/07/scaling), and [talked about the collaborative peer review that JOSS promotes](https://www.youtube.com/watch?v=niRiyaErqwQ).

## How to apply

Firstly, we especially welcome applications from prospective editors who will contribute to the diversity (e.g., ethnic, gender, disciplinary, and geographical) of our board. 

✨✨✨ If you're interested in applying please fill in this [short form](https://forms.gle/SwNqHd3KYi1rwPMEA) by 9th November 2024. ✨✨✨

## Who can apply

Applicants should have significant experience in open source software and software engineering, as well as expertise in at least one subject/disciplinary area. Prior experience with peer-review and open science practices are also beneficial.  

We are especially interested in recruiting editors in the following disciplines, although submissions from all disciplines are welcome:

> - Both Pure and Applied Mathematics

Reviewer: ```suggestion
- Pure and/or Applied Mathematics
```

- Bioinformatics
- Computational Chemistry

> - Computer Science

Reviewer: ```suggestion
- Computer Science
- Quantum Computing
```

Reviewer: I don't know what ordering we are using now... I would keep these together topically, if possible, but they don't have to be...

## Who can apply

Applicants should have significant experience in open source software and software engineering, as well as expertise in at least one subject/disciplinary area. Prior experience with peer-review and open science practices are also beneficial.  

We are especially interested in recruiting editors in the following disciplines, although submissions from all disciplines are welcome:

- Pure and/or Applied Mathematics
- Bioinformatics
- Computational Chemistry
- Computer Science
- Quantum Computing
- Ecology
- Economics
- Geology
- Image Processing
- Language Processing
- Physics & Engineering: Energy Systems
- Physics & Engineering: Fluid Dynamics
- Remote Sensing

The JOSS editorial team has a diverse background and there is no requirement for JOSS editors to be working in academia.

### Expected level of committment 

> Editing for JOSS is a substantial committment, and we would encourage you to read our [editorial guide](https://joss.readthedocs.io/en/latest/editing.html), and in particular our [expectations of editors](https://joss.readthedocs.io/en/latest/expectations.html). While the guide has a bunch more information, some things we think are particulatly important include 1) pay ongoing attention to the submissions they are handling (i.e., checking in on papers ~twice per week for a few minutes at a time), 2) responding in a timely fashion to invites to edit, and 3) contributing to the overall editorial team (e.g., participating in scope reviews, giving feedback to colleagues in Slack).

Author: ```suggestion
Editing for JOSS is a substantial commitment, and we would encourage you to read our [editorial guide](https://joss.readthedocs.io/en/latest/editing.html), and in particular our [expectations of editors](https://joss.readthedocs.io/en/latest/expectations.html). While the guide has a bunch more information, some things we think are particularly important include 1) paying ongoing attention to the submissions they are handling (i.e., checking in on papers ~twice per week for a few minutes at a time), 2) responding in a timely fashion to invites to edit, and 3) contributing to the overall editorial team (e.g., participating in scope reviews, giving feedback to colleagues in Slack).
```
